.gitignore

```diff
@@ -25,3 +25,7 @@ deltachat-ffi/html
 deltachat-ffi/xml
 
 .rsynclist
+
+# ignore backup files
+*.bak
```

Comment:
Makes sense, since we already ignore `*.db`, though it is not directly related to fix.